ui/imports/utils/LocaleUtils.qml

```diff
@@ -19,4 +25,20 @@ QtObject {
 
         return num.toLocaleString(locale, 'f', precision)
     }
+
+    function currencyAmountToLocaleString(currencyAmount, locale) {
+        if (!locale) {
+            console.log("Unspecified locale for: " + JSON.stringify(currencyAmount))
+            locale = Qt.locale()
+        }
+        var amount = parseFloat(currencyAmount.amount)
+        var amountStr = numberToLocaleString(amount, currencyAmount.displayDecimals, locale)
+        if (currencyAmount.stripTrailingZeroes) {
+            amountStr = stripTrailingZeroes(amountStr, locale)
+        }
+        if (currencyAmount.symbol) {
+            amountStr += " " + currencyAmount.symbol
```

Comment:
Almost there :) How do you know the symbol goes after the number?

Reply:
John said so :D
See the end of https://github.com/status-im/status-desktop/issues/8640

We're not using the actual symbol ( $, €, £) since it's not specific enough, we're using what seems to be interchangeably referred to as symbol/code (USD, EUR, GBP) and we've decided to always put it after the number.

Reply:
Ok, fair enough. What about right to left languages?

Reply:
At the minimum, the whole thing should use String.arg()

Reply:
> Ok, fair enough. What about right to left languages?

I'll ask John, no special comments were made about that.

> At the minimum, the whole thing should use String.arg()

Will do

Reply:
> At the minimum, the whole thing should use String.arg()

Done